pkgs/development/tools/stem/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchurl
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
```suggestion
python3Packages.buildPythonPackage rec {
```

Reply:
`python3.pkgs` just aliases to `python3packages`, so it should be exactly the same. I'm not sure if there's a style guide though.